src/com/mendix/widget/carousel/components/CarouselItem.ts

```diff
@@ -6,7 +7,7 @@ export interface CarouselItemProps {
 }
 
 export const CarouselItem = (props: CarouselItemProps) => (
-    DOM.div({ className: "item" + (props.active ? " active" : "") },
+    DOM.div({ className: classnames({ active: props.active, item: true }) },
```

Comment:
=> `classNames("item", { active: props.active })`

Two reasons, first this is shorter. Second, item is always there so put that first (common case first).